host/4/bullseye/amd64/dotnet/dotnet-inproc/install_ca_certificates.sh

```diff
@@ -3,15 +3,33 @@
 # Source and destination directories
 source_dir="/var/ssl/root"
 destination_dir="/usr/local/share/ca-certificates"
+need_certificate_update=false
+
+if [[ "$WEBSITES_INCLUDE_CLOUD_CERTS" == "true" ]]; then
```

Comment:
@CooperLink - is the Env var "$WEBSITES_INCLUDE_CLOUD_CERTS" expected to be set when running in AGC?